src/openms/source/ANALYSIS/ID/AccurateMassSearchEngine.cpp

```diff
@@ -677,6 +690,17 @@ namespace OpenMS
       results_part[hit_idx].setObservedRT(feature.getRT());
       results_part[hit_idx].setSourceFeatureIndex(feature_index);
       results_part[hit_idx].setObservedIntensity(feature.getIntensity());
+      
+      std::vector<double> mti;
+      for (Size i=0; i<3; ++i)
+      {
+        if (feature.metaValueExists("masstrace_intensity_" + String(i)))
```

Comment:
So wouldnt the correct way to store this be in sub-features? Also, I think this is because I added the option to store FFMetabo output in consensusXML because there the full intensity traces of a masstrace can be stored

I agree, but wouldnt then make sense to store in a meta value how many isotopic traces there are? instead of just assuming there are 3

I guess for this we would have to adopt the upstream tool , but would it make sense to have the number of reported traces to be controlled by the user?